remedy.lic

```diff
@@ -17,11 +17,14 @@ class Remedy
 
   def initialize
     @settings = get_settings
+    @hometown = @settings.hometown
```

Comment:
What would you think about something like `@hometown = @settings.force_remedies_crafting_town || @settings.force_crafting_town || @settings.hometown`? Do you think that makes the order of precedence clearer?

Reply:
I liked the symmetry with how the other scripts work (and in some cases HAS to work due to how smith works) to keep the code logic as close to the same, but am not opposed to switching it up if it's more in line with ruby style standards.

Reply:
It's not a big deal